data/03-09-random-forest.php

```diff
@@ -44,14 +44,62 @@
-			<p>O princípio básico do NB consiste em treinar um modelo que possibilite a aplicação do <a href="https://pt.wikipedia.org/wiki/Teorema_de_Bayes"><strong>Teorema de Bayes</strong> (ou regra de Bayes)</a> para estimar a classe mais provável de um novo objeto. Para tal, na etapa de treinamento o algoritmo computa uma tabela de probabilidades que resume a base de dados de treinamento com informações suficientes para a aplicação deste teorema.</p>
-			<p>A seguir apresentaremos um exemplo que mostra detalhadamente o processo de treinamento de um classificador NB. Considere a base de dados apresentada na Tabela 4. Suponha que as 15 observações desta base representem dados de uma pesquisa sobre o endividamento de casais jovens. A base registra se o casal possui filhos, a escolaridade do(a) responsável pela família e se a família possui ou não algum tipo de dívida com o cartão de crédito.</p>
-
-			<!-- Tabela [inicio] -->
+		<div class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
+			<p>Suponha que seu computador está travando com frequência e você deseja descobrir o motivo. Em vez de chamar apenas um técnico para examiná-lo, você poderia optar por chamar vários. Neste caso, se uma determinada causa para o problema for mais citada do que qualquer outra, você a consideraria como a diagnóstico correto. Ou seja, a causa do problema de travamento seria determinada de acordo com uma espécie de votação majoritária, onde cada técnico daria o seu voto. A sua expectativa é que considerar a opinião mais prevalente de um grupo de técnicos é mais confiável do que levar em conta a opinião de um único profissional.</p>
+			<p>A Floresta Aleatória (<i>Random Forest</i> – RF) é uma técnica de classificação que trabalha com uma ideia similar. Ela combina as decisões de múltiplos modelos para otimizar a eficácia do processo de classificação. Mais especificamente, uma RF é um <strong>modelo de classificação composto</strong>, que é formado por um conjunto <i>(ensemble)</i> de classificadores do tipo árvore de decisão. Cada árvore de decisão do grupo é gerada com o uso de subconjuntos independentes e aleatórios de atributos e objetos. Para prever a classe de um novo objeto, cada classificador individual vota e a classe mais popular é devolvida ao usuário.</p>
+			<p>A Figura 10 resume o funcionamento de um classificador RF. Nas subseções a seguir, utilizaremos essa figura como base para explicar as etapas de treinamento e classificação da RF.</p>
+		</div>	
+			
+		<div class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
+			<h2>ETAPA DE TREINAMENTO</h2>
+			<p>Um modelo RF combina uma série de k modelos de árvore de decisão, AD1, AD2, ..., ADk, com o objetivo de criar um modelo composto otimizado de classificação. Cada árvore de decisão individual é também chamada de classificador base <i>(base classifier)</i>.</p>
```

Comment:
As variaveis AD1, AD2 etc tem o indice subescrito.
Pra isso vc precisa usar a tag `<sub>`

```
AD<sub>1</sub>, AD<sub>2</sub>, ..., AD<sub>k</sub>
```

Reply:
O k em ...combina uma série de k models de árvore... tem itálico

Reply:
Ao invés de `<i>(base classifier)</i>` deveria ser `(<i>base classifier</i>)`. Os parênteses ficam "fora" do tálico.

Reply:
Toda vez que vc tiver uma sigla em maiúsculas, tipo em: Um modelo RF vc precisa colocar a sigla em `small-caps`.
Fica: ```Um modelo <span class="small-caps">RF</span>```